src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,124 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
+    private static String DB_NAME = "internet_shop";
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public Item create(Item item) {
+        String name = item.getName();
+        Double price = item.getPrice();
+        String query = String.format("INSERT INTO %s.items (name, price) VALUES('%s', %.0f)",
+                DB_NAME, name, price);
+        try (Statement stmt = connection.createStatement()) {
+            int rs = stmt.executeUpdate(query);
+            logger.info("Created row(s) :" + rs);
+        } catch (SQLException e) {
+            logger.error("Connection failed", e);
```

Comment:
```suggestion
            logger.error("Can't create item", e);
```
please, be more informative while creating logs

Reply:
or you can throw a RuntimeException instead